todo_app/data/trello_items.py

```diff
@@ -0,0 +1,51 @@
+import os, requests, json
+
+key = os.environ.get('API_KEY')
```

Comment:
Blocker - these environment variables should be added to the `.env.template` and to the README, along with some explanation about how to get these variables.